### PR TITLE
Bump enlightenment to 0.21.9

### PR DESCRIPTION
**Pull request by optarg**

### CODE REVIEW

File: components/desktop/e/enlightenment/Makefile [hunk 1]
```diff
@@ -45,6 +45,7 @@ CONFIGURE_BINDIR.64=$(USRBINDIR)
 CONFIGURE_OPTIONS+=	--sysconfdir=/etc
 CONFIGURE_OPTIONS+=	--enable-shared
 CONFIGURE_OPTIONS+=	--disable-static
+CONFIGURE_OPTIONS+= --disable-cpufreq
```
Comment (optarg) [inline components/desktop/e/enlightenment/Makefile]: This module is a no-op on illumos. It currently only supports Linux sysfs and *BSD sysctl.

Comment (pyhalov) [inline components/desktop/e/enlightenment/Makefile]: please, fix formatting

Comment (pyhalov) [reply components/desktop/e/enlightenment/Makefile]: And what is wrong with it?

Comment (pyhalov) [reply components/desktop/e/enlightenment/Makefile]: OK

Comment (optarg) [reply components/desktop/e/enlightenment/Makefile]: Done.

**Pull request merged**